example/ndpi_util.c

```diff
@@ -458,12 +458,16 @@ static struct ndpi_flow_info *get_ndpi_flow_info(struct ndpi_workflow * workflow
       workflow->stats.ndpi_flow_count++;
 
       *src = newflow->src_id, *dst = newflow->dst_id;
+      
+      if(workflow->hrs == 1)newflow->n_pckt_human_readable_string++;
```

Comment:
Protocols like SSL in certificates have human readable strings. How do you deal with that?